test/functional/core/job_spec.lua

```diff
@@ -666,13 +666,16 @@ describe("pty process teardown", function()
   end)
 
   it("does not prevent/delay exit. #4798 #4900", function()
-    if helpers.pending_win32(pending) then return end
     -- Use a nested nvim (in :term) to test without --headless.
-    feed_command(":terminal '"..helpers.nvim_prog
-      .."' -u NONE -i NONE --cmd '"..nvim_set.."' "
+    feed_command(":terminal "
+      ..(iswin() and helpers.nvim_prog or "'"..helpers.nvim_prog.."'")
+      .." -u NONE -i NONE --cmd '"..nvim_set.."' "
```

Comment:
Why did it fail in Appveyor?

```
[00:10:12] [  ERROR   ] C:/projects/neovim/test/functional\core\job_spec.lua @ 668: pty process teardown does not prevent/delay exit. #4798 #4900

[00:10:12] .\test\functional\ui\screen.lua:302: Row 1 did not match.

[00:10:12] Expected:

[00:10:12]   |*^                              |

[00:10:12]   |[Process exited 0]            |

[00:10:12]   |                              |

[00:10:12]   |                              |

[00:10:12]   |                              |

[00:10:12]   |                              |

[00:10:12] Actual:

[00:10:12]   |*^Error detected while processin|

[00:10:12]   |g pre-vimrc command line:     |

[00:10:12]   |E20: Mark not set             |

[00:10:12]   |Press ENTER or type command to|

[00:10:12]   | continue                     |

[00:10:12]   |                              |

[00:10:12] 

[00:10:12] To print the expect() call that would assert the current screen state, use

[00:10:12] screen:snapshot_util(). In case of non-deterministic failures, use

[00:10:12] screen:redraw_debug() to show all intermediate screen states.  

[00:10:12] 

[00:10:12] stack traceback:

[00:10:12] 	.\test\functional\ui\screen.lua:302: in function 'wait'

[00:10:12] 	.\test\functional\ui\screen.lua:216: in function 'expect'

[00:10:12] 	C:/projects/neovim/test/functional\core\job_spec.lua:681: in function <C:/projects/neovim/test/functional\core\job_spec.lua:668>
```

Reply:
Unsetting shada does nothing. I don't know what's causing `E20`.